tests/test_omegaconf.py

```diff
@@ -469,3 +469,40 @@ def test_is_issubclass() -> None:
     cfg = OmegaConf.structured(ConcretePlugin)
     t = OmegaConf.get_type(cfg)
     assert t is not None and issubclass(t, ConcretePlugin)
+
+
+@mark.parametrize(
+    ("cfg", "expected"),
+    [
+        # dict
+        param({}, {}, id="dict"),
+        param({"a": 10, "b": "${a}"}, {"a": 10, "b": 10}, id="dict"),
+        param({"a": 10, "b": {"a": "${a}"}}, {"a": 10, "b": {"a": 10}}, id="dict"),
+        param({"a": "${b.a}", "b": {"a": 10}}, {"a": 10, "b": {"a": 10}}, id="dict"),
+        param({"a": "???"}, {"a": "???"}, id="dict:missing"),
+        param({"a": "???", "b": "${a}"}, {"a": "???", "b": "???"}, id="dict:missing"),
+        param({"a": 10, "b": "a_${a}"}, {"a": 10, "b": "a_10"}, id="dict:str_inter"),
+        # This seems like a reasonable resolution for a string interpolation pointing to a missing node:
```

Comment:
Yeah, I was going to mention it in the code, but catching `InterpolationToMissingValueError` will turn string / nested interpolations into missing values. That's ok with me.